lib/search-summary/index.js

```diff
@@ -65,24 +57,6 @@ export default class SearchSummary extends Component {
           />
         </div>
       );
-    } else if (visibleTab === 'edit-details') {
-      return (
-        <div className='searchSummaryContainer'>
-          <EditDetails
-            {...this.props}
-            exitButtonClick={this.closeTab}
-            onSearchClick={() => { this.setState({visibleTab: 'header'}); startSearch(); }}
-            changeRoute={changeRoute}
-            goBack={goBack}
-          />
-        </div>
-      );
-    } else if (visibleTab === 'tags') {
-      return (
-        <div className='searchSummaryContainer'>
-          <TagView exitButtonClick={this.closeTab}/>
-        </div>
-      );
     } else {
       return <div/>;
     }
```

Comment:
Can you clean up this component? There are lots of proptypes which are no longer being used.  The `visibleTab` state can also be change so it is just `visible: true` or `visible: false`.